go.mod

```diff
@@ -1,32 +1,37 @@
 module go.temporal.io/sdk
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
 
 require (
 	github.com/facebookgo/clock v0.0.0-20150410010913-600d898af40a
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.6.0
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
+	github.com/nexus-rpc/sdk-go v0.0.8-0.20240502185337-2b47041a2cc2
```

Comment:
I know we're not merging this PR until server tagged, but would also recommend this Nexus project is tagged before any sdk-go release with this dependency

Reply:
Sure, I can add this to a checklist.

Reply:
Done.